### ISSUE TITLE
Measure the draft and weekly gates' ceilings, so S6's NOT-RUNNABLE print points at its remedy

### ISSUE THREAD

**Issue by jacksonmlukas** [jacksonmlukas/football-hub#376]
## What this is

The measurement #363 (S6, option 1 -- adopted 2026-09-20) makes necessary. Under option 1 a gate that hands in no ceiling returns NOT-RUNNABLE in both directions, and two of the three gates have never handed one in: `docs/gate-power.md:289` -- *"the two gates that have never measured one still reach the verdicts they reached before."* Both already carry the mechanism (`draft.backtest --ceiling` via `with_ceiling`, `weekly_gate --ceiling` via `declared_ceiling`); neither has been **run** with it.

The ADR-0019 amendment S6 writes says *"NOT-RUNNABLE until this ticket measures the ceilings"* and cites this number, so the print names what ends it rather than becoming a second unnamed permanent NOT-RUNNABLE state (rule 16; #300 named the first).

## Not frozen

A measurement, not a modelling change. It changes no model, no constant, no published number.

## Acceptance criteria

- [ ] The **weekly** gate run once with `--ceiling`; its ceiling (the foresight arm's diff, per `declared_ceiling`) recorded on this ticket and in `docs/gate-power.md` beside the gate's MDE, so the NOT-RUNNABLE branch can read it
- [ ] The **draft** gate run once with `--ceiling` at the default `n_drafts=20`; same recording. This is a second full simulation pass and may land in a separate commit -- a commit that lands one of the two says `Advances #<this>` and names the one still owed, so `scripts/partial_work.sh` surfaces it
- [ ] Neither run overwrites `state/gate-width.json` on `main` before #362 lands: #362's acceptance is *which run produced each entry and whether it was published*, and the current file **is** that evidence. A pre-run copy is committed as `state/gate-width.2026-09-20.pre-ceiling.json`; after a run, `state/gate-width.json` is restored and the run's width goes on this ticket
- [ ] Closes only when both ceilings are recorded

## Blocked by

- Nothing. Weekly is cheap and runs first; draft waits for a machine that is not also running #368's 4,000-trial leverage study.

---

Filed 2026-09-20 in the grilling session that adopted #363; the disposition is the maintainer's.

**Comment by jacksonmlukas** [comment]
Weekly gate's ceiling, measured -- weekly half only tonight (draft, S6's other half, still owed; it contends for CPU with #368's leverage study).

`uv run python -m hub.season.weekly_gate --run --ceiling` (seasons 2022-2025, drafts=20, seed 0, frozen rosters, restricted, mask_pool):

| | this run |
|---|---|
| weekly - consensus | -1.156 |
| 95% CI, percentile | [-1.761, -0.259] |
| clusters | 3, not 4 |
| season-clustered MDE (80% power) | 1.926 |
| **ceiling (perfect foresight)** | **+10.799** |
| verdict | REMOVE |

**Stage 2 passes, by about 5.6x** (10.799 / 1.926): `experiment.gate`'s NOT-RUNNABLE branch does not fire for the weekly gate. Full writeup, including a flag on why this run only scored 3 clusters (season 2022 produced no row) instead of the default --seasons's 4, in `docs/gate-power.md` beside the gate's MDE.

`state/gate-width.json` is unchanged: the ceiling run's write to it (`weekly.clusters: 3, width: 1.503`) was reverted with `git checkout -- state/gate-width.json` immediately after, so #362's evidence stays what it was before tonight. The pre-run copy is `state/gate-width.2026-09-20.pre-ceiling.json`, unchanged and identical to it.

Advances this ticket; closes only when the draft half lands too.

**Comment by jacksonmlukas** [comment]
The weekly ceiling above is a **3-cluster** figure: season 2022 was absent from the run's paired frame while the published gate has four held-out seasons. Filed as #378 (bug). Until it resolves, the ADR-0019 amendment's weekly rule-16 row is withheld and this ceiling is read at k = 3 only.

**Comment by jacksonmlukas** [comment]
**Weekly half corrected under #378.** This comment's own `--ceiling` run scored 3 clusters
because season 2022 dropped silently -- root cause and fix in #378, closed by this lane.

Root cause: `weekly_gate.compare`'s walk-forward split (`experiment.expanding_seasons`) always
trains each scored season on every season strictly before it, so the *earliest* season handed
to it is consumed as training data and never itself scored. `docs/weekly-blend-gate.md`'s
Reproduce section already names five seasons for this reason -- a sacrificial buffer ahead of
the four held-out ones. The `--ceiling` run above instead used `weekly_gate`'s own `--seasons`
default, which named exactly the four held-out seasons with nothing ahead of them, so
`expanding_seasons` dropped 2022 -- the earliest of *those four* -- rather than a season
nobody meant to score. Nothing was wrong with reading this gate at k=4; the run was reading an
accidental k=3.

Fixed: `weekly_gate`'s `--seasons` default now names the buffer season
(`2021,2022,2023,2024,2025`), and `hub.season.weekly_gate_data.SeasonDropped` refuses a run
outright if any season goes missing from the scored output beyond that one expected buffer
drop. Every run now also prints what it asked for beside what it scored.

Re-run at the corrected default, same recipe otherwise (drafts=20, seed 0, frozen rosters,
restricted, mask_pool):

| | this thread's run (wrong) | **corrected** |
|---|---|---|
| weekly − consensus | −1.156 | **−1.147** |
| 95% CI, percentile | [−1.761, −0.259] | **[−1.605, −0.556]** |
| clusters | 3, not 4 | **4** |
| season-clustered MDE (80% power) | 1.926 | **1.119** |
| **ceiling (perfect foresight)** | +10.799 | **+10.873** |
| ceiling / MDE | 5.6x | **9.7x** |

Stage 2 still passes, and by a wider margin than the mistaken run reported: the corrected MDE
is *smaller*, because a genuine fourth season narrows the interval a dropped-season run
cannot. The weekly gate is not the underpowered one, at the k its own docs and the ADR-0019
amendment's rule-16 table both name -- that row is unstruck as of #378, unchanged, since it
was computed at k=4 all along.

This run's own verdict (SHOW under the post-S1 tie-aware rule, not the REMOVE the 3-cluster
run printed) is a property of this ceiling-measurement recipe -- no `--shrink`, `drafts=20` --
and is not `docs/weekly-blend-gate.md`'s published −1.004 REMOVE, which this ticket does not
re-run and does not move. Full writeup in `docs/gate-power.md`'s dated 2026-09-21 restatement
beside this comment's own table.

`state/gate-width.json` restored after the re-run (`git checkout -- state/gate-width.json`),
same as this thread's original run.

**Comment by jacksonmlukas** [comment]
Draft half measured -- the second, owed half of this ticket.

`uv run python -m hub.draft.backtest --seasons 2022,2023,2024,2025 --drafts 20 --seed 0 --ceiling` (the module's own documented recipe, `--ceiling` added, default `n_drafts=20`; this gate has no `--shrink` flag, that belongs to `weekly_gate`):

| | this run |
|---|---|
| optimizer - market | -12.40 |
| 95% CI, percentile | [-17.60, -7.20] |
| 95% CI, t | [-21.11, -3.69] |
| clusters | 4 |
| season-clustered MDE (80% power) | 11.01 |
| **ceiling (perfect foresight)** | **+21.90** |
| **ceiling / MDE** | **~1.99x** |
| verdict, this run | SHOW (tie-aware rule: won 0, tied 1, lost 3 of 4 seasons) |

**Stage 2 passes -- MDE (11.01) sits below the ceiling (21.90), by about 1.99x.** Thinner than the weekly half's 9.7x, but clear: `experiment.gate`'s NOT-RUNNABLE branch does not fire for the draft gate either. Both season gates ADR-0019's #363 (S6) amendment named NOT-RUNNABLE pending this ticket now carry a measured ceiling.

This run's own SHOW verdict is a property of this ceiling-measurement recipe (`n_drafts=20`, no shrink) and is not the published ADR-0009 REMOVE, which this ticket does not re-run and does not move. Full writeup, including the ADR-0019 rule-16 cross-reference the coordinator asked for (stage-2 guard passing vs. the ADOPT branch's separately-measured 0.0324 combined power at delta=2.0 -- runnable is not the same as able to adopt), is in `docs/gate-power.md`'s dated 2026-09-21 restatement beside the weekly note, and in `docs/adr/0019-a-gate-requires-every-season.md`'s rule-16 section.

`state/gate-width.json` is unchanged: the run's own write (`draft`, clusters 4, **width 10.403535714285717**, verdict SHOW, timestamp 2026-09-21T18:24:02+00:00) was reverted with `git checkout -- state/gate-width.json` immediately after recording, so #362's evidence stays what it was before tonight. `state/gate-width.2026-09-20.pre-ceiling.json` is the byte-identical pre-run copy, unchanged, predating both halves.

Flagged, not chased down (out of scope): this run also printed `REQUIRES REVIEW` -- its season-clustered interval (width 10.40) is narrower than the previously recorded one (11.10, ratio 0.94), the shape #169 found unnoticed in five of nine cases.

Both halves recorded. Closing.

**Comment by jacksonmlukas** [comment]
**Diagnosis: why the season-clustered draft-gate width moved 11.10 -> 10.40 (`REQUIRES REVIEW`).**

The 11.10 entry (`lo -18.761, hi -7.665`) is commit `7e6625f` (2026-09-16), the record for ADR-0009's #290 hold-out run at `9855979`: `--holdout --workers 4`, seed 0, 20 drafts x 4 seasons, replaying each season under `conf/holdout/<season>.json` with `WEEKLY_K` and `TEAMMATE_RHO` refit leave-one-season-out.

Tonight's run (`--seasons 2022,2023,2024,2025 --drafts 20 --seed 0 --ceiling`) carried **no `--holdout`**, so `compare()` played every season on shipped constants instead -- **the recipes do not match.** `--ceiling` itself never touches the interval: it only hands `run_gate` a foresight bound carried beside the summary (`experiment.summarise`'s `carried = {} if ceiling is None else {"ceiling": ceiling}`); the bootstrap producing `mean`/`lo`/`hi`/`width` runs only over `compare()`'s `paired` frame, which reads `holdout=a.holdout` -- `False` tonight.

A second, independent mover: of every `fitted`/`chosen` constant on the draft path (`TALENT_CV`, `WEEKLY_K`, `WEEKLY_SKEW`, `MARGIN_SD`, `TEAMMATE_RHO`, `PICK_NOISE_INTERCEPT`, `PICK_NOISE_SLOPE`), only `IMPUTE_CV`/`IMPUTE_CV_BY_POS` moved between `9855979` and `main` -- commit `02561b7`, 32 minutes after the hold-out run (#298): pooled 0.260 -> 0.315 (QB 0.217->0.315, RB 0.334->0.359, WR 0.223->0.288, TE 0.220->0.315). `hub.holdout`'s `HELD_OUT` set never overrides `IMPUTE_CV` either way (recorded "not refitted"), so this move applies regardless of `--holdout` and is not explained by the flag alone.

No other declared constant, the #197 frozen-Board test pin, or `board.py`/`backtest.py`'s own `fitted`/`chosen` lines moved between the two commits. The live Board each run rebuilds from `data/` also drifts with nflverse backfills independent of code (ADR-0009 already documents this for its own re-runs) and is not re-checked here (CLAUDE.md rule 1).

**Conclusion: recipe mismatch (`--holdout` vs. not) plus one already-tracked constant move (#298), not a defect.** No bug filed; no re-run performed. Full writeup: `docs/gate-power.md`, dated note beside the 2026-09-21 draft-ceiling block's `REQUIRES REVIEW` line.
